spec/index.bs

```diff
@@ -3916,15 +3920,27 @@ dictionary GPURasterizationStateDescriptor {
     GPUDepthBias depthBias = 0;
     float depthBiasSlopeScale = 0;
     float depthBiasClamp = 0;
+
+    GPUSize32 sampleCount = 1;
+    GPUSampleMask sampleMask = 0xFFFFFFFF;
```

Comment:
If sample-rate shading is enabled, I assume this takes effect before the fragment shader, and not after? (In which case this is an okay place for it, though I think it doesn't technically affect rasterization.)

Reply:
@kvark said:
> the sample mask is applied at the rasterizer stage unless the post-depth coverage is explicitly enabled, which I don't think we have